src/electrum/electrs.cpp

```diff
@@ -14,6 +14,21 @@
 
 static std::string monitoring_port() { return GetArg("-electrum.monitoring.port", "4224"); }
 static std::string monitoring_host() { return GetArg("-electrum.monitoring.host", "127.0.0.1"); }
+static std::string rpc_host() { return GetArg("-electrum.host", "127.0.0.1"); }
+static std::string rpc_port(const std::string &network)
+{
+    std::map<std::string, std::string> portmap = {{"main", "50001"}, {"test", "60001"}, {"regtest", "60401"}};
+
+    auto defaultPort = portmap.find(network);
+    if (defaultPort == end(portmap))
```

Comment:
I'm confused as to how this code can handle a number like "24", yet I see you are testing this case

Reply:
The portmap stuff is just to figure out what default to use _if -electrum.port_ is not set